schedule/yast/yast2_gui/yast2_gui_sle.yaml

```diff
@@ -73,6 +74,12 @@ conditional_schedule:
         - yast2_gui/yast2_datetime
         - yast2_gui/yast2_hostnames
 test_data:
+  net_device: eth0
+  errors_in_ifcfg_file:            # Each command is followed by ifcfg file name.
+#   - "sed -i 's/dhcp/dgcp/'"      # Typo in value. To be enabled once bsc#1181296 is fixed.
```

Comment:
this line looks like commented, I think is some git visualization, perhaps it is better to put the comments on top.

Reply:
Yes, it is on purpose, see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/11839#pullrequestreview-575212985

Reply:
Could you please file progress ticket for this, so we don't forget to enable this check?